Assets/Tests/InputSystem/Plugins/HIDTests.cs

```diff
@@ -1040,12 +1040,7 @@ public void TODO_Layouts_CanBuildCustomLayoutsBasedOnTheHIDLayoutBuilder()
     [Category("Utilities")]
     public void Utilities_CanRecognizeVendorDefinedUsages()
     {
-        string usagePage;
-        string usage;
-
-        HID.UsageToString((HID.UsagePage) 0xff01, 0x33, out usagePage, out usage);
-
-        Assert.That(usagePage, Is.EqualTo("Vendor-Defined"));
-        Assert.That(usage, Is.EqualTo("Vendor-Defined"));
+        Assert.That(HID.UsagePageToString((HID.UsagePage) 0xff01), Is.EqualTo("Vendor-Defined"));
+        Assert.That(HID.UsageToString((HID.UsagePage) 0xff01, 0x33), Is.Null);
```

Comment:
Note this change in behavior and API. I figured that returning null would be more consistent then returning "Vendor-Defined", when we return null in all other cases of "we don't know the usages for this page".

Reply:
(Crap, hit resolve accidentally)